ExtendedExample does not work with enums

The example fails to run with the following stack trace:
```java
Exception in Application start method
java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at javafx.graphics/com.sun.javafx.application.LauncherImpl.launchApplicationWithArgs(LauncherImpl.java:465)
	at javafx.graphics/com.sun.javafx.application.LauncherImpl.launchApplication(LauncherImpl.java:364)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.base/sun.launcher.LauncherHelper$FXHelper.main(LauncherHelper.java:1051)
Caused by: java.lang.RuntimeException: Exception in Application start method
	at javafx.graphics/com.sun.javafx.application.LauncherImpl.launchApplication1(LauncherImpl.java:901)
	at javafx.graphics/com.sun.javafx.application.LauncherImpl.lambda$launchApplication$2(LauncherImpl.java:196)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.reflect.InaccessibleObjectException: Unable to make field public static final com.dlsc.preferencesfx.demo.extended.ExtendedExample$MyEnum com.dlsc.preferencesfx.demo.extended.ExtendedExample$MyEnum.FIRST accessible: module com.dlsc.preferencesfx.demo does not "opens com.dlsc.preferencesfx.demo.extended" to module com.google.gson
	at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:340)
	at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:280)
	at java.base/java.lang.reflect.Field.checkCanSetAccessible(Field.java:176)
	at java.base/java.lang.reflect.Field.setAccessible(Field.java:170)
	at com.google.gson@2.8.9/com.google.gson.internal.bind.TypeAdapters$EnumTypeAdapter$1.run(TypeAdapters.java:746)
	at com.google.gson@2.8.9/com.google.gson.internal.bind.TypeAdapters$EnumTypeAdapter$1.run(TypeAdapters.java:744)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at com.google.gson@2.8.9/com.google.gson.internal.bind.TypeAdapters$EnumTypeAdapter.<init>(TypeAdapters.java:744)
	at com.google.gson@2.8.9/com.google.gson.internal.bind.TypeAdapters$28.create(TypeAdapters.java:790)
	at com.google.gson@2.8.9/com.google.gson.Gson.getAdapter(Gson.java:489)
	at com.google.gson@2.8.9/com.google.gson.Gson.toJson(Gson.java:727)
	at com.google.gson@2.8.9/com.google.gson.Gson.toJson(Gson.java:714)
	at com.google.gson@2.8.9/com.google.gson.Gson.toJson(Gson.java:669)
	at com.google.gson@2.8.9/com.google.gson.Gson.toJson(Gson.java:649)
	at com.dlsc.preferencesfx@11.9.0/com.dlsc.preferencesfx.util.StorageHandlerImpl.serialize(StorageHandlerImpl.java:27)
	at com.dlsc.preferencesfx@11.9.0/com.dlsc.preferencesfx.util.PreferencesBasedStorageHandler.loadObject(PreferencesBasedStorageHandler.java:227)
	at com.dlsc.preferencesfx@11.9.0/com.dlsc.preferencesfx.model.Setting.loadSettingValue(Setting.java:517)
	at com.dlsc.preferencesfx@11.9.0/com.dlsc.preferencesfx.model.PreferencesFxModel.lambda$loadSettingValues$7(PreferencesFxModel.java:213)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
	at com.dlsc.preferencesfx@11.9.0/com.dlsc.preferencesfx.model.PreferencesFxModel.loadSettingValues(PreferencesFxModel.java:209)
	at com.dlsc.preferencesfx@11.9.0/com.dlsc.preferencesfx.PreferencesFx.init(PreferencesFx.java:70)
	at com.dlsc.preferencesfx@11.9.0/com.dlsc.preferencesfx.PreferencesFx.<init>(PreferencesFx.java:65)
	at com.dlsc.preferencesfx@11.9.0/com.dlsc.preferencesfx.PreferencesFx.<init>(PreferencesFx.java:56)
	at com.dlsc.preferencesfx@11.9.0/com.dlsc.preferencesfx.PreferencesFx.of(PreferencesFx.java:103)
	at com.dlsc.preferencesfx.demo/com.dlsc.preferencesfx.demo.extended.ExtendedExample.createPreferences(ExtendedExample.java:166)
	at com.dlsc.preferencesfx.demo/com.dlsc.preferencesfx.demo.extended.ExtendedExample.<init>(ExtendedExample.java:92)
	at com.dlsc.preferencesfx.demo/com.dlsc.preferencesfx.demo.AppStarter.start(AppStarter.java:26)
	at javafx.graphics/com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$9(LauncherImpl.java:847)
	at javafx.graphics/com.sun.javafx.application.PlatformImpl.lambda$runAndWait$12(PlatformImpl.java:484)
	at javafx.graphics/com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:457)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at javafx.graphics/com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:456)
	at javafx.graphics/com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:96)
	at javafx.graphics/com.sun.glass.ui.win.WinApplication._runLoop(Native Method)
	at javafx.graphics/com.sun.glass.ui.win.WinApplication.lambda$runLoop$3(WinApplication.java:184)
	... 1 more
Exception running application com.dlsc.preferencesfx.demo.AppStarter

Process finished with exit code 1
```

The last few lines of the log are:
> 01.06.2022 15:36:58.909 [JavaFX Application Thread] TRACE com.dlsc.preferencesfx.model.PreferencesFxModel - Loading: Appearance & Behaviour#null#Project Opening
01.06.2022 15:36:58.909 [JavaFX Application Thread] TRACE com.dlsc.preferencesfx.model.PreferencesFxModel - Loading: Appearance & Behaviour#Synchronization#Synchronize files on frame or editor tab activation
01.06.2022 15:36:58.910 [JavaFX Application Thread] TRACE com.dlsc.preferencesfx.model.PreferencesFxModel - Loading: Appearance & Behaviour#Synchronization#Save files on frame deactivation
01.06.2022 15:36:58.910 [JavaFX Application Thread] TRACE com.dlsc.preferencesfx.model.PreferencesFxModel - Loading: Appearance & Behaviour#Synchronization#Saves files automatically if application is idle for 15 sec.
01.06.2022 15:36:58.910 [JavaFX Application Thread] TRACE com.dlsc.preferencesfx.model.PreferencesFxModel - Loading: Appearance & Behaviour#Synchronization#Use safe write (save changes to a temporary file first)
01.06.2022 15:36:58.910 [JavaFX Application Thread] TRACE com.dlsc.preferencesfx.model.PreferencesFxModel - Loading: Appearance & Behaviour#Accessibility#Support screen readers (requires restart)
01.06.2022 15:36:58.910 [JavaFX Application Thread] TRACE com.dlsc.preferencesfx.model.PreferencesFxModel - Loading: Appearance & Behaviour#null#On closing Tool Window with Running Process
01.06.2022 15:36:58.910 [JavaFX Application Thread] TRACE com.dlsc.preferencesfx.model.PreferencesFxModel - Loading: Appearance & Behaviour#null#My Enum

I had to comment out:
https://github.com/dlsc-software-consulting-gmbh/PreferencesFX/blob/dec39ef49704bde5ab0d39c9e8b4f3fd2c2ca5ca/preferencesfx-demo/src/main/java/com/dlsc/preferencesfx/demo/extended/ExtendedExample.java#L298-L299
To get it to run.

--- a/preferencesfx-demo/src/main/java/module-info.java
+++ b/preferencesfx-demo/src/main/java/module-info.java
@@ -8,4 +8,5 @@ module com.dlsc.preferencesfx.demo {
     exports com.dlsc.preferencesfx.demo;
 
     opens com.dlsc.preferencesfx.demo;
+    opens com.dlsc.preferencesfx.demo.extended;
 }

